x-pack/metricbeat/module/aws/ec2/_meta/fields.yml

```diff
@@ -68,3 +68,39 @@
       type: long
       description: >
         Reports whether the instance has passed the instance status check in the last minute.
+    - name: instance.core.count
+      type: integer
+      description: >
+        The number of CPU cores for the instance.
+    - name: instance.image.id
+      type: keyword
+      description: >
+        The ID of the image used to launch the instance.
+    - name: instance.monitoring.state
+      type: keyword
+      description: >
+        Indicates whether detailed monitoring is enabled.
+    - name: instance.private.dns_name
+      type: keyword
+      description: >
+        The private DNS name of the network interface.
+    - name: instance.private.ip
+      type: keyword
+      description: >
+        The private IPv4 address associated with the network interface.
+    - name: instance.public.dns_name
+      type: keyword
+      description: >
+        The public DNS name of the instance.
+    - name: instance.public.ip
+      type: keyword
```

Comment:
Why is this keyword and not `ip` type? Also for the other ip field above?

Reply:
@ruflin sorry that's a mistake... I just pushed a PR to fix this. Thanks!